lego/apps/events/constants.py

```diff
@@ -10,6 +10,7 @@
 SOCIAL = "social"
 OTHER = "other"
 EVENT = "event"
+IMAGE = "image"
```

Comment:
What's this used for?

Reply:
Already defined somewhere else, removed and fixed import.